back-end/app.js

```diff
@@ -208,4 +208,43 @@ app.use((err, req, res, next) => {
   res.status(500).send({ error: err });
 });
 
+// PATCH endpoint to update card metadata
+app.patch("/card/:cardId", (req, res, next) => {
+  const cardId = req.params.cardId;
```

Comment:
```suggestion
  const {cardId} = req.params;
```
same as last PR, minor but looks more concise to me